contracts/Dixel.sol

```diff
@@ -122,17 +122,22 @@ contract Dixel is Ownable, ReentrancyGuard, DixelSVGGenerator {
                 prevY = params[i].y;
 
                 Pixel storage pixel = pixels[params[i].x][params[i].y];
-                uint200 oldPrice = pixel.price;
+                // compare old color with incoming color
+                if(pixel.color != params[i].color) {
```

Comment:
I guess we better just through an error here right a way instead of looping through all parameters

```solidity
require(pixel.color != params[i].color, "UNCHANGED_PIXEL_COLORS") 
```